src/ralph/accounts/admin.py

```diff
@@ -61,7 +62,7 @@ def url(self, item):
     url.title = _('Link')
 
 
-class UserInfoView(RalphDetailView):
+class UserInfoView(RalphDetailView, metaclass=PermissionViewMetaClass):
```

Comment:
Because other class inherit from RalphDetailView and we have conflict (example: RalphDetailViewAdmin)